AutomatedTesting/Gem/PythonTests/editor/EditorScripts/ComponentCRUD_Add_Delete_Components.py

```diff
@@ -125,7 +125,8 @@ async def add_component(component_name):
         success = await pyside_utils.wait_for_condition(lambda: not hydra.has_components(entity_id, ['Mesh']), 5.0)
         Report.result(Tests.mesh_component_deleted, success)
 
-        # 7) Undo deletion of component
+        # 7) Undo deletion of component after waiting for the deletion to register
+        general.idle_wait(0.5)
```

Comment:
alternatively you could use general.idle_wait_frames(1)

the undo could also be done with general.undo()

Reply:
Required more than 1 frame of wait, or I would have used wait_for_propagation, but it doesn't appear to be prefab propagation it's waiting on here, and the pyside usage is to more closely simulate user input.

Reply:
If it requires more than 1 frame of wait, shouldn't it use "idle_wait_frames(N)" instead of an absolute time then? If there's a system hiccup it seems like an absolute time won't guarantee that more than 1 frame has passed.

Reply:
> If it requires more than 1 frame of wait, shouldn't it use "idle_wait_frames(N)" instead of an absolute time then? If there's a system hiccup it seems like an absolute time won't guarantee that more than 1 frame has passed.

Good point. I'll play with it and see about updating to a specific frame value.

Reply:
Was able to update this to use wait_for_propagation. A previous portion of the test that was waiting on an Entity Inspector refresh was updated to wait for 3 frames.